src/subScreen/basalRatesScreen.js

```diff
@@ -0,0 +1,67 @@
+const match = require('../match');
+const action = require('../action');
+const config = require('../config');
+
+class BasalRatesScreen {
+    constructor(language) {
+        this.language = language;
+    }
+    Header() {
+        return match.accessible.Header(this.language.settingsScreen.BasalRates);
+    }
+    BackButton() {
+        return match.accessible.BackButton(this.language.settingsScreen.Settings);
+    }
+    SaveButton() {
+        return match.accessible.Label(this.language.settingsScreen.SaveToSimulator);
+    }
+    AddButton() {
+        return match.accessible.ButtonBarButton(this.language.general.Add);
+    }
+    async Save() {
+        await this.SaveButton().tap();
+    }
+    async Close() {
+        await this.BackButton().tap();
+    }
+    /**
+     * @summary basal rate to be set. NOTE: it is assumed that the rates are given in order of time
+    * @param {Object} rate
+     * @param {String} rate.time
+     * @param {String} rate.unitsPerHour
+     */
+    async Apply(rate) {
+        if (rate) {
+            await this.AddButton().tap();
+            await match.accessible.Label(`${rate.time}`).atIndex(0).tap();
+            await action.SetPickerValue(1, `${rate.unitsPerHour} ${config.basalRatesUnits}`);
+        }
+    }
+    /**
+     * @summary basal rates to be set. NOTE: it is assumed that the rates are given in order of time
```

Comment:
[nit] Consider sorting `rates` on `time` to validate? Also does this imply that `rates[0].time` must also be `0`?